case_studies/sdss_galaxies/plots.py

```diff
@@ -594,6 +595,7 @@ def compute_data(self, frame: Union[SDSSFrame, SimulatedFrame], encoder, decoder
             w_end = w + scene_size
             true = frame.image[:, :, h:h_end, w:w_end]
             coadd_params = frame.get_catalog((h, h_end), (w, w_end))
+            coadd_params.plocs -= 0.5
```

Comment:
I would suggest doing this adjustment within your plotting code rather than mutating the FullCatalog.

Reply:
Thanks, that makes sense.